examples/golang/bucket-notification-deletion/notification-deletion-example/notification-deletion-example.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+func main() {
+	bucket := flag.String("b", "", "Name of the bucket to add notification to")
+	topic := flag.String("t", "", "The topic onto which the notification is attached to")
```

Comment:
why do we need a topic here?
we should have an optional "notificationID" indicating deletion of specific notification or all notifications of a bucket